source/Glimpse.Site/Areas/api/Controllers/VersionController.cs

```diff
@@ -1,23 +1,33 @@
 ﻿using System;
 using System.Web.Http;
+using System.Collections.Generic;
 using Glimpse.Package;
 using Glimpse.Site.Framework;
+using Microsoft.ApplicationInsights;
+
 
 namespace Glimpse.Site.Areas.api.Controllers
-{ 
+{
     [HttpHeader("Access-Control-Allow-Origin", "*")]
     public class VersionController : ApiController
-    { 
+    {
+        private TelemetryClient telemetry = new TelemetryClient();
```

Comment:
I know this is picky but just from a style perspective can there be a new line between the private variables and the action?